src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -119,10 +152,15 @@ public class CoinstakeContext
         // Default for -blockmintxfee, which sets the minimum feerate for a transaction in blocks created by mining code
         public const int DefaultBlockMinTxFee = 1000;
 
-        // Default for -blockmaxsize, which controls the maximum size of block the mining code will create
+        // Default for -blockmaxsize, which controls the maximum number of serialized bytes in a block the mining code will create.
         public const int DefaultBlockMaxSize = 750000;
 
-        // Default for -blockmaxweight, which controls the range of block weights the mining code will create
+        /// <summary>
+        /// Default for -blockmaxweight, which controls the maximum of block weight the mining code will create.
```

Comment:
I'd prefer having those switches inside "" or alike. maybe even `<c>`, but "" would be good enough imho

Reply:
also pls check if that can be changed elsewhere